docs/rhoas/rhoas-service-contexts/README.adoc

```diff
@@ -237,11 +237,15 @@ The Quarkus application uses a topic in a Kafka instance to produce and consume
 In addition, the producer and consumer components serialize and deserialize Kafka messages using an Avro schema stored in {registry}. Use of the schema ensures that message values conform to a defined format.
 
 .Prerequisites
+
+ifndef::community[]
+* You have a {org-name} account.
+endif::[]
 * You have a service context with a Kafka and {registry} instance.
 * https://github.com/git-guides/[Git^] is installed.
 * You have an IDE such as https://www.jetbrains.com/idea/download/[IntelliJ IDEA^], https://www.eclipse.org/downloads/[Eclipse^], or https://code.visualstudio.com/Download[Visual Studio Code^].
-* https://adoptopenjdk.net/[OpenJDK^] 11 or later is installed on Linux or MacOS. (The latest LTS version of OpenJDK is recommended.)
-* https://maven.apache.org/[Apache Maven^] 3.8.x or later is installed (for Quarkus 2.2.x).
+* https://adoptopenjdk.net/[OpenJDK^] 11 or later is installed. (The latest LTS version of OpenJDK is recommended.)
+* https://maven.apache.org/[Apache Maven^] 3.8.x or later 3.x is installed.
```

Comment:
Same comment as https://github.com/redhat-developer/app-services-guides/pull/598/files#r1044277775.

Unless I'm misreading this, it looks like a mistake?

Reply:
Thanks, John -- the intention is to show that it is only later 3.x versions that are supported, not 4.x or later. Would it help if I inserted the word "version", and perhaps in parentheses?
 - "3.8.x or later 3.x version is installed"
 - "3.8.x (or later 3.x version) is installed"

Reply:
Ahhh, I see. I think the second reformulation (using parentheses) makes that much clearer :+1: I might state it as "**a** later 3.x version....", however.

Reply:
One remaining comment is that maybe we're overusing `x` here? Do we want to consider [semantic versioning](https://semver.org/) instead? That is:

```
* https://maven.apache.org/[Apache Maven^] 3.8.z (or a later 3.y version) is installed.
```
I don't know - perhaps I'm overthinking it. WDYT?

Reply:
Actually, I see that according to IBM Style, we shouldn't use any such notation! How about the following rewrite instead?

`* https://maven.apache.org/[Apache Maven^] 3.8 (or a later Maven 3 release) is installed.`

Reply:
Ah, that's a good thing to know! Yes, I think that rewrite looks great.